examples/guestbook-go/_src/main.go

```diff
@@ -27,11 +27,12 @@ import (
 	"github.com/xyproto/simpleredis"
 )
 
-var pool *simpleredis.ConnectionPool
+var masterPool *simpleredis.ConnectionPool
```

Comment:
This should be written as

var(
  masterPool *simpleredis.ConnectionPool
  slavePool *simpleredis.ConnectionPool
)